Content.Client/Audio/AmbientSoundSystem.cs

```diff
@@ -127,56 +113,122 @@ private void ClearSounds()
             _playingSounds.Clear();
         }
 
-        /// <summary>
-        /// Get a list of ambient components in range and determine which ones to start playing.
-        /// </summary>
-        private void SampleNearby(EntityCoordinates coordinates)
+        private (Dictionary<string, List<AmbientSoundComponent>>, HashSet<EntityUid>) GetNearbySources(EntityCoordinates coordinates)
         {
-            var compsInRange = new List<AmbientSoundComponent>();
+            var sourceDict = new Dictionary<string, List<AmbientSoundComponent>>();
+            var set = new HashSet<EntityUid>();
 
-            foreach (var entity in _lookup.GetEntitiesInRange(coordinates, _maxAmbientRange,
-                LookupFlags.Approximate | LookupFlags.IncludeAnchored))
+            foreach (var entity in _lookup.GetEntitiesInRange(coordinates, _maxAmbientRange + RangeBuffer, LookupFlags.IncludeAnchored))
```

Comment:
seems like this will be slower now that it isnt approximate

Reply:
Removing approximate was me trying to debug the insanity that is it returning inconsistent results.